Pseudocode/Sorting/Insertion Sort/SourceCode.tex

```diff
@@ -0,0 +1,65 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[]{algpseudocode}
+
+% Manually remove the 'end' for some sections
+\algtext*{EndFunction}
+\algtext*{EndWhile}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+% Define the command for a boldface 'and' 'to'
+\newcommand{\To}{\textbf{ to }}
+\newcommand{\And}{\textbf{ and }}
+% Use them inside Math-Mode (Hence the space!)
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Sort an array by repeatedly inserting elements into a sorted sequence}
+
+  \begin{algorithmic}[1]
+    \Ensure One Based Indexing for the Array
+    \Require A non empty array
+    \Statex
+
+    \Function{$Insertion\_Sort$}{$arr$}
```

Comment:
Just include a `LeftComment` here by `\LeftComment{\textbf{Loop Invariant} : $arr[1 \dots j-1]$ is sorted}`.

By here, I mean after the function signature.